linux_os/guide/system/auditing/auditd_configure_rules/directory_permissions_var_log_audit/tests/common_0700.sh

```diff
@@ -0,0 +1,9 @@
+# platform = multi_platform_ol,multi_platform_rhel
+#!/bin/bash
+
```

Comment:
I think you needs to add `# packages = audit` to the test scenarios because the scenarios use files coming from the audit RPM package.